monai/data/utils.py

```
@@ -120,7 +120,7 @@ def get_random_patch(
     """
 
     # choose the minimal corner of the patch
-    rand_int = np.random.randint if rand_state is None else rand_state.randint
+    rand_int = np.random.RandomState().randint if rand_state is None else rand_state.randint
     min_corner = tuple(rand_int(0, ms - ps + 1) if ms > ps else 0 for ms, ps in zip(dims, patch_size))
```

Ruff (0.15.7)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)